Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -153,6 +153,15 @@ def audit
       else
         puts "#{f.full_name}:", problem_lines.map { |s| "  #{s}" }
       end
+
+      next unless ENV["GITHUB_ACTIONS"]
```

Comment:
Might be nice to have a `GitHub::Actions.something?` boolean method for this.

Reply:
Will keep this in mind, I don't think it's worth adding this yet. Might be possible to come up with a nicer API in future like

```ruby
Github::Actions.annotate do
  error "message", …
end
```